lib/livebook_space_web/plug.ex

```diff
@@ -0,0 +1,26 @@
+defmodule LivebookSpaceWeb.Plug do
+  use Plug.Builder
+
+  plug Plug.Static,
+    from: {:livebook_space, "priv/static/iframe"},
+    at: "/iframe",
+    # Iframes are versioned, so we cache them for long
+    cache_control_for_etags: "public, max-age=31536000",
+    headers: [
+      # Enable CORS to allow Livebook fetch the content and verify its integrity
+      {"access-control-allow-origin", "*"},
+      {"content-type", "text/html; charset=utf-8"}
+    ]
+
+  plug Plug.Static, from: :livebook_space, at: "/"
+
+  plug :not_found
+
+  defp not_found(%{request_path: "/", method: "GET"} = conn, _) do
```

Comment:
We typically match on path_info because it is normalized. For example, `request_path` may be `//`, which is normalized into a single `/` on `path_info`: 

```suggestion
  defp not_found(%{path_info: [], method: "GET"} = conn, _) do
```